docs/UserGuide.adoc

```diff
@@ -341,7 +342,100 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
-== Features (Coming in v2.0)
+== Features(Coming in v2.0)
+
+=== Add a relationship
+
+Adds a relationship between two persons. +
+Format: `addRelationship INDEXOFSOURCE INDEXOFDESTINATION DIRECTION`
+
+****
+* Add a relationship between the person with `INDEXOFSOURCE` and `INDEXOFDESTINATION`, +
+with the direction of the relationship specified.
+* `INDEXOFSOURCE` refers to the index of the person from whom the relationship is +
+initiated as shown in the most recent listing.
+* `INDEXOFDESTINATION` refers to the index of the person to whom the relationship is directed.
+* `DIRECTION` referring to the direction of this relationship, can only be String input of `directed` or `undirected`.
+* If the `DIRECTION` is `directed`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` matters.
+* If the `DIRECTION` is `undirected`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` does not matter.
+* The index *must be a positive integer* `1, 2, 3, ...`
+****
+
+Examples:
+
+* `list` +
+`addRelationship 2 3 directed` +
+Adds a directed relationship starting from the 2nd person to the 3rd person in the list.
+* `list` +
+`addRelationship 2 3 undirected` +
+Adds an undirected relationship between the 2nd person and the 3rd person in the list.
+
+=== Add a named relationship
+
+Adds a relationship between two persons and gives this relationship a name. +
+Format: `addNamedRelationship INDEXOFTHESOURCE INDEXOFTHEDESTINATION DIRECTION NAME`
+
+****
+* Adds a relationship between `INDEXOFTHESOURCE` and `INDEXOFTHEDESTINATION` +
+with the direction of the relationship specified and name of the relationship given.
+* `INDEXOFSOURCE` refers to the index of the person from whom the relationship is +
+initiated as shown in the most recent listing.
+* `INDEXOFDESTINATION` refers to the index of the person to whom the relationship is directed.
+* `DIRECTION` referring to the direction of this relationship, can only be String input of `directed` or `undirected`.
+* `NAME` referring to the name of the relationship *can only be alphanumeric*.
+* If the `DIRECTION` is `directed`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` matters.
+* If the `DIRECTION` is `undirected`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` does not matter.
+* The index *must be a positive integer* `1, 2, 3, ...`
+****
+
```

Comment:
Change this section as per the suggestions for `addRelationship` since it is pretty much the same